### ISSUE TITLE
load-from-code-file (js schema) throws error on windows machine

### ISSUE THREAD

**Issue by kroeder** [dotansimha/graphql-code-generator#2875]
**Describe the bug**
Using this codegen.yml

```yml
schema: "./projects/next-client-mock-schema/src/generate/schema-js.js"
overwrite: true
generates:
  ./graphql.schema.json:
    plugins:
      - "introspection"
```
throws 

```
 × ./graphql.schema.json
    Failed to load schema from ./projects/next-client-mock-schema/src/generate/schema-js.js:

            Unable to load from file "C:\path-to-project\projects\next-client-mock-schema\src\generate\schema-js.js": Cannot find module 'C:_myplace
    extlibs@aeb
    ext-clientprojects
    ext-client-mock-schemasrcgenerateschema-js.js'
            Error: Unable to load from file "C:\path-to-project\projects\next-client-mock-schema\src\generate\schema-js.js": Cannot find module 'C:_myplace
    extlibs@aeb
    ext-clientprojects
    ext-client-mock-schemasrcgenerateschema-js.js'
        at tryToLoadFromExport (C:\path-to-project\node_modules\graphql-toolkit\src\loaders\load-from-code-file.ts:84:11)
        at Object.loadFromCodeFile (C:\path-to-project\node_modules\graphql-toolkit\src\loaders\load-from-code-file.ts:116:20)
    
            GraphQL Code Generator supports:
              - ES Modules and CommonJS exports (export as default or named export "schema")
              - Introspection JSON File
              - URL of GraphQL endpoint
              - Multiple files with type definitions (glob expression)
              - String in config file
    
            Try to use one of above options and run codegen again.
    Error: Failed to load schema
        at Object.exports.loadSchema (C:\path-to-project\node_modules\@graphql-codegen\cli\src\load.ts:80:11)
    Error: Failed to load schema
        at Object.exports.loadSchema (C:\path-to-project\node_modules\@graphql-codegen\cli\src\load.ts:80:11)
```

The schema file worked perfectly running `< 1.0.0` 

I started debugging the CLI and ended up at this line:
`node_modules\graphql-toolkit\dist\commonjs\loaders\load-from-code-file.js`

```javascript
async function tryToLoadFromExport(rawFilePath) {
    let filePath = rawFilePath;
    try {
        if (require && require.cache) {
            filePath = eval(`require.resolve('${filePath}')`);
            if (require.cache[filePath]) {
                delete require.cache[filePath];
            }
        }
        const rawExports = await eval(`require('${filePath}');`); // <--- this is causing trouble
```

After changing it to

```javascript
        const rawExports = await eval(`require('${filePath.split('\\').join('\\/')}');`);
```

my output is 

```
> graphql-codegen --config codegen.yml

  √ Parse configuration
  √ Generate outputs
Running lifecycle hook "afterStart" scripts...
[CLI] Loading Schemas
[CLI] Loading Documents
[CLI] Generating output
```
and the generated schema looks as expected.

**To Reproduce**

I'm not entirely sure how to reproduce this.
**Some facts:**
- I'm on a Windows machine 
- I generate a schema.json using a js file

**The js file contains**
```
require('ts-node').register({ transpileOnly: true });
module.exports = require('./schema.ts').default;
```

**I tried this and it worked without changing load-from-code-file.js**
```
const { buildSchema } = require('graphql');

module.exports = buildSchema(/* GraphQL */ `
  type MyType {
    foo: String!
  }

  type Query {
    myType: MyType!
  }
`);
```

It might fail because I added ts-node on top of my current solution.
My change
```javascript
        const rawExports = await eval(`require('${filePath.split('\\').join('\\/')}');`);
```
did the trick for me but I'm not sure if it affects other machines in other scenarios 

If you think this is an appropriate solution I'd be happy to contribute this, just let me know.

**Expected behavior**
No error

**Environment:**

- OS: Windows
- `@graphql-codegen/...`: 1.8.2
- NodeJS: 10.16.0

**Comment by kroeder** [comment]
I just figured out this is related to graphql-toolkit, not graphql-code-generator (although it affects @graphql-code-generator)

It will be even harder for me to describe the bug in graphql-toolkit without knowing what happens under the hood of graphql-code-generator

Can you assist solving this/raising an issue in graphql-toolkit?

**Comment by ardatan** [comment]
@kroeder Could you try with the following canary version?
`1.8.4-alpha-40191ca5.21+40191ca5`

**Comment by kroeder** [comment]
Works like a charm 🙂

**Comment by dotansimha** [comment]
Fixed in 1.9.0
